extensions/wikia/ArticleVideo/styles/recommended-video.scss

```diff
@@ -0,0 +1,219 @@
+@import 'skins/oasis/css/core/layout';
+@import 'skins/oasis/css/core/breakpoints-variables';
+
+.article-recommended-video-unit {
+	background-color: rgba(255, 255, 255, 0.98);
+	bottom: 0;
+	box-shadow: 0 2px 9px 0 rgba(0, 0, 0, 0.5);
+	box-sizing: border-box;
+	display: flex;
+	font-size: 14px;
+	justify-content: center;
+	left: 0;
+	min-width: 718px;
+	padding: 15px 0 40px;
+	position: fixed;
+	right: 0;
+	transform: translate(0, 1000px);
+	transition: 0.3s transform ease-in;
+	z-index: 10000;
+
+	&__close-button {
+		align-items: center;
+		cursor: pointer;
+		display: flex;
+		height: 24px;
+		justify-content: center;
+		position: absolute;
+		right: 10px;
+		top: 10px;
+		width: 24px;
+	}
+
+	&.revealed {
+		transform: translate(0, 0);
+	}
+
+	&.expanded {
+		height: auto;
+
+		.actual-video {
+			flex-grow: 1;
+			flex-shrink: 1;
+			flex-basis: 678px;
+			order: 1;
+
+			h3 {
+				font-size: 24px;
+				font-weight: 500;
+				margin-bottom: 26px;
+				margin-top: 24px;
+			}
+		}
+
+		.side-playlist {
+			display: flex;
+			flex: 0;
+			flex-basis: 400px;
+		}
+
+		.video-placeholder {
+			display: none;
+		}
+
+		.side-playlist .video-placeholder {
+			display: flex;
+		}
+
+		.side-playlist .article-recommended-video {
+			display: flex;
+			flex-grow: 1;
+			width: auto;
+		}
+
+		.article-recommended-video-placeholder {
+			min-width: 122px;
+			width: 122px;
+		}
+	}
+}
+
+.article-recommended-videos-wrapper {
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	max-width: calc(100vw - 2 * #{$padding-width-small});
+
+	@if $widthType == 0 {
+		@media #{$breakpoint-small-only} {
+			width: $content-width-small;
+		}
+
+		@media #{$breakpoint-small-plus} {
+			width: $content-width-small-plus;
+		}
+
+		@media #{$breakpoint-medium-only} {
+			width: $content-width-medium;
+		}
+
+		@media #{$breakpoint-large-up} {
+			width: $content-width-large;
+		}
+	}
+
+	h2 {
+		align-self: flex-start;
+		color: #000;
+		font-size: 14px;
+		margin-bottom: 16px;
+		text-transform: uppercase;
+
+		strong {
+			font-weight: bold;
+		}
+	}
+
+	.side-playlist {
+		display: none;
+		flex-direction: column;
+		margin-left: 47px;
+		order: 2;
+
+		h3 {
+			margin-left: 12px;
+			margin-top: 0;
+		}
+
+		.video-placeholder {
+			align-items: center;
+			margin-bottom: 18px;
+		}
+
+		svg {
+			height: 30px;
```

Comment:
you don't need to set sizes here, just add `wds-player-icon-play-tiny` class to svg element